<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>640</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>640</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>640</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="loading">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>640</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/loading/Android Small - 1.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButt_logo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/login/Group 4.png</normaloff>image/login/Group 4.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sign_in">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>640</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/login/Android Small - 11.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButt_enter_sign_in">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>350</y>
        <width>281</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="enter_number_sign_in">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButt_sign_up">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="get_flname_driver">
     <widget class="QTextEdit" name="fname">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="lname">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>290</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButt_next_get_flname_driver">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>281</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>640</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/flname_driver/Android Small -.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButt_back_get_flname_driver">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>281</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>fname</zorder>
     <zorder>lname</zorder>
     <zorder>pushButt_next_get_flname_driver</zorder>
     <zorder>pushButt_back_get_flname_driver</zorder>
    </widget>
    <widget class="QWidget" name="select_driver_user">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>640</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/select_driver_user/Android Small - 7.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButt_select_user">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButt_select_driver">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="accept_code_sign_up">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>640</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/send_code/Android Small - 5.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButt_accept_sign_up_code">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="accept_code_sign_in">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>640</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/send_code/Android Small - 5.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButt_accept_sign_in_code">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sign_up">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>640</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/signin/Android Small - 8.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButt_enter_sign_up">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>281</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButt_sign_in">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="enter_number_sign_up">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
